tests/NuGet.config

```diff
@@ -0,0 +1,11 @@
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+      <!--
+      Contains RemoteExecutor. Taken from:
+      https://github.com/dotnet/runtime/blob/master/NuGet.config
+      -->
+    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
```

Comment:
You can add this feed directly into the `RestoreSources` property in the solution root `Directory.Build.Props`. The config file can then be deleted.